draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -1026,21 +1030,22 @@ document's "site for cookies".
 
 Shared workers may be bound to multiple documents at once. As it is quite
 possible for those documents to have distinct "site for cookie" values, the
-worker's "site for cookies" will be the empty string in cases where the values
-diverge, and the shared value in cases where the values agree.
+worker's "site for cookies" will be an origin set to a globally unique 
+identifier in cases where the values diverge, and the shared origin in cases 
+where the values agree.
```

Comment:
Previously, this would be the shared site (registered domain), whereas now they need to all be exactly the same origin. (The algorithm below seems fine though.)

Reply:
Do you have any concerns with this text as-is, or just pointing out the difference?

Reply:
Sorry for being unclear. I was pointing out that this doesn't match the actual algorithm below. I think we can just replace "where the values diverge" with "where the values are not all same-site with the worker's origin", and replace "the shared origin" with "the worker's origin", in cases where the values are all same-site with it.

Reply:
Ah, now I see. Thanks!